src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -598,5 +598,23 @@ public void TestLargeTextWriterDoesNotResuseSmallChunks()
             Assert.Equal(1, GetChunks(newText).Length);
             Assert.NotSame(chunk1, GetChunks(newText)[0]);
         }
+
+        [Fact]
+        [WorkItem(10452, "https://github.com/dotnet/roslyn/issues/10452")]
+        public void TestEmptyChangeAfterChange()
+        {
+            SourceText.From("Hello World")
+                .WithChanges(new TextChange(new TextSpan(5, 6), string.Empty)) // prepare a ChangedText instance
+                .WithChanges(); // this should not cause exception
```

Comment:
Should the test also assert what the content is? I agree not throwing is a good bar, but not quite sufficient for quality!